model/user.go

```diff
@@ -142,6 +143,9 @@ func AuthenticateUser(identifier string, password []byte) (user User, ok bool) {
 
 	// The parameters hash and password are replaced with an empty slice by
 	// the validPassword function.
+	log.Println(string(hash))
+	log.Println(string(password))
```

Comment:
Well... It's... HIGHLY important that you remove these.

Reply:
WOOOPS
![WOOPS](https://media.giphy.com/media/5hmlX9levfAC6kbBRk/giphy.gif)